### DISCUSSION TITLE
Disable Custom Highlight with Checkbox

### DISCUSSION THREAD

**Discussion by 2547techno** [Chatterino/chatterino2#5285] [Ideas]
Currently, the only way to disable a custom highlight is to delete the highlight entirely, while the default highlights can be toggled with a checkbox.

The suggestion is to add a checkbox for custom highlights as well so they can be disabled without deletion.

![image](https://github.com/Chatterino/chatterino2/assets/109011672/4879bf43-5f40-40dd-8dac-75cc08898607)

**Comment by Mm2PL** [comment]
Just add !NO or something in the meantime or a regex that never matches anything like `^$disabled highlight text goes here`.
